test/e2e/framework/util.go

```diff
@@ -4770,3 +4777,72 @@ func DumpDebugInfo(c clientset.Interface, ns string) {
 		Logf("\nLast 100 log lines of %v:\n%v", s.Name, l)
 	}
 }
+
```

Comment:
test/e2e/common might be better... I don't really know the organizing principle anymore.